stko/calculators/open_babel_calculators.py

```diff
@@ -60,17 +64,25 @@ def __init__(self, forcefield):
 
         """
 
+        if openbabel is None:
+            raise WrapperNotInstalledException(
+                'openbabel is not installed; see README for '
+                'installation.'
+            )
+
         self._forcefield = forcefield
 
     def calculate(self, mol):
         temp_file = 'temp.mol'
         mol.write(temp_file)
-        obConversion = openbabel.OBConversion()
-        obConversion.SetInFormat("mol")
-        OBMol = openbabel.OBMol()
-        obConversion.ReadFile(OBMol, temp_file)
-        OBMol.PerceiveBondOrders()
-        os.system('rm temp.mol')
+        try:
+            obConversion = openbabel.OBConversion()
+            obConversion.SetInFormat("mol")
+            OBMol = openbabel.OBMol()
+            obConversion.ReadFile(OBMol, temp_file)
+            OBMol.PerceiveBondOrders()
```

Comment:
Do we want an additional Except statement here?

Reply:
Nah, if it fails it should raise an openbabel error and should not be excepted. It is the file read in.